tests/quantization/test_torchao.py

```diff
@@ -227,11 +227,9 @@ def test_reload_weights():
     "currently https://github.com/pytorch/ao/issues/2919, we'll have to skip "
     "torchao tests that requires newer versions (0.14.0.dev+) for now"
 )
-def test_opt_125m_float8_weight_only_safetensors_model_loading_with_params(vllm_runner):
+def test_safetensors_model_loading_with_params(vllm_runner):
     torch._dynamo.reset()
-    model_name = (
-        "torchao-testing/opt-125m-Float8WeightOnlyConfig-v2-0.14.0.dev-safetensors"
-    )
+    model_name = "torchao-testing/Qwen3-8B-INT4-0.15.0dev-safetensors"
```

Comment:
Why are you using this model? If it is for "different tensor attributes for one instance of a tensor subclass (ie qdata or scale for Float8Tensor) being sharded to different files.", please add a comment.